### PR TITLE
Websocket / Subscription Manual Added

### PR DESCRIPTION
**Pull request by Wouter125**
We never really covered on how to use our GraphQL subscriptions. Heck, our websocket URL wasn't even described! This PR aims to solve that issue. It still needs to curl examples to be added, so will work on that next.

### CODE REVIEW

File: content/Getting-Started/1+API-Basics/subscriptions.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Communicating on the websocket
+With all configuration done you're finally ready to send subscriptions and receive useful data. This is almost identical to the authorization, but with one exception! You'll need to add `id` set to `1` to your message. Every next message will be an increment of 1 so nobody can cut the line. To give you an idea of how this looks, we'll be using our [routeUpdatedById](/API-Reference/Routes/subscribe-to-route-updates) subscription as an example, and only requesting its processing `status`. 
+
+<code-block lang="bash" prefix="Subscriptions" title="Communicating">
+{ "id": "1", "type": "start", "payload": { "query": "subscription routeUpdatedById { routeUpdatedById(id: \"Your route id here\") { status } }", "variables": { "id": "Your route id here"} } }
+</code-block>
+
+If everything was correct and you set the `routeId`, the websocket will respond with an object that contains the `status`. If you didn't set your id, there will be **no** response from the server.
+
+Last but not least; if you got the result you were looking for, **don't forget to close the connection!**
```
Comment (timkuilman) [inline content/Getting-Started/1+API-Basics/subscriptions.md:43]: Please add an explanation of how you can close the connection.

File: content/Getting-Started/1+API-Basics/subscriptions.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# Subscriptions
+
+The Chargetrip API relies on GraphQL technology, so some requests use a [subscription](https://spec.graphql.org/draft/#sec-Subscription) mechanism. Unlike queries or mutations, subscriptions are long-lasting operations and can change their result over time. These operations are handled by a websocket. Authenticating and communicating with this websocket is different than a regular query or mutation. Let's go over it in detail.
+
+<note display="block">
+Websocket connections can't be used out of the box in your terminal. So if you would like to tag along with the steps below you can use a library called wscat. It's open source and can be downloaded [here](https://github.com/websockets/wscat).
+</note>
+
+## Library usage
+The guide below is for those not opting in on a library like [Apollo GraphQL](https://www.apollographql.com/), and rather taking care of it manually. If you do opt in for such a library, please follow along with the tutorials on the libraries website. They already take care of certain configurations for you.
```
Comment (timkuilman) [inline content/Getting-Started/1+API-Basics/subscriptions.md:16]: and rather take car off it manually (singular? )

File: content/Getting-Started/1+API-Basics/subscriptions.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+If everything was correct, you'll receive the following message from the websocket; `{ "type": "connection_ack" }`.
+
+## Communicating on the websocket
+With all configuration done you're finally ready to send subscriptions and receive useful data. This is almost identical to the authorization, but with one exception! You'll need to add `id` set to `1` to your message. Every next message will be an increment of 1 so nobody can cut the line. To give you an idea of how this looks, we'll be using our [routeUpdatedById](/API-Reference/Routes/subscribe-to-route-updates) subscription as an example, and only requesting its processing `status`. 
+
+<code-block lang="bash" prefix="Subscriptions" title="Communicating">
+{ "id": "1", "type": "start", "payload": { "query": "subscription routeUpdatedById { routeUpdatedById(id: \"Your route id here\") { status } }", "variables": { "id": "Your route id here"} } }
+</code-block>
+
+If everything was correct and you set the `routeId`, the websocket will respond with an object that contains the `status`. If you didn't set your id, there will be **no** response from the server.
```
Comment (timkuilman) [inline content/Getting-Started/1+API-Basics/subscriptions.md:41]: You also will not receive a message when the route computation already finished before the connection is acknowledge. This is why we recommend to always query the route once.

**Review by timkuilman — DISMISSED** [review]
See my comments. In docs I don't really like the usage of contractions like you'll, we're, you're. I checked the Getting Started section in the dev portal and noticed we write 'you are' in full. Let's be consistent with that.

File: content/Getting-Started/1+API-Basics/subscriptions.md (hunk 3 above)
Comment (Wouter125) [reply content/Getting-Started/1+API-Basics/subscriptions.md:41]: That's specific to the route query, we are just using it as an example. I think we highlight this specific issue in the route section and the route introduction.

File: content/Getting-Started/1+API-Basics/subscriptions.md (hunk 2 above)
Comment (Wouter125) [reply content/Getting-Started/1+API-Basics/subscriptions.md:16]: Those is plural. Also Neelesh already reviewed it.

"For those rather taking care of it manually."

File: content/Getting-Started/1+API-Basics/subscriptions.md (hunk 1 above)
Comment (Wouter125) [reply content/Getting-Started/1+API-Basics/subscriptions.md:43]: Check good suggestion, will do.

**Comment by Wouter125** [comment]
Alright will get rid of the contractions.

File: content/Getting-Started/1+API-Basics/subscriptions.md (hunk 3 above)
Comment (timkuilman) [reply content/Getting-Started/1+API-Basics/subscriptions.md:41]: I agree that this is a generic websocket introduction and that we already explain this in the route section but if people follow these instructions they might be surprised that no status is returned even though an id is set. (We'll return an error soon in that scenario). 

What about changing it to: 


If everything was correct and you set the `routeId`, the websocket will respond with an object that contains `status` whenever the value of that field is updated. For more information about the route subscription have a look at the [route section](https://developers.chargetrip.com/API-Reference/Routes/introduction). If you didn't set your id, there will be **no** response from the server.

**Review by timkuilman — APPROVED** [review]
LGTM

**Pull request merged**